k2/python/k2/fsa.py

```diff
@@ -36,58 +37,59 @@ class Fsa(object):
 
     An instance of FSA has the following attributes:
 
-    arcs
-      You will NOT use it directly in Python. It is an instance of
-      ``_k2.RaggedArc`` with only one method ``values()`` which
-      returns a 2-D `torch.Tensor`` of dtype ``torch.int32`` with 4
-      columns. Its number of rows indicates the number of arcs in the
-      FSA. The first column represents the source states, second
-      column the destination states, third column the labels and the
-      fourth column is the score. Note that the score is actually
-      a float number but it is **reinterpreted** as an integer.
+    - ``arcs``: You will NOT use it directly in Python. It is an instance
```

Comment:
Could you use the previous one as it is better rendered in the documentation?

Reply:
Sorry merged so I could debug.. hopefull this can be fixed later...

Reply:
OK，will do later, seems it's from merging stuff somehow (or local auto format), I never edit those documentation except aux_labels